test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/Crc32Tests.cs

```diff
@@ -40,5 +41,19 @@ public void Crc32_DifferentOutputForDifferentData()
             // Assert
             code1.Should().NotBe(code2);
         }
+
+        [Fact]
+        public void CalculateCrc_WithZeroBytes_CrcXorsToMagicNumber()
```

Comment:
Can you please tell me what's the purpose of this test?

Reply:
Since we now own a CRC-32 implementation, we should verify it's correct.  The ZIP specification (see the comment in the test) specifies a "magic number" for the CRC.  This test proves that this implementation verifies against the magic number.